app/src/main/java/com/example/hangouts/homeScreen/fragments/HangoutDetailsFragment.java

```diff
@@ -0,0 +1,132 @@
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class HangoutDetailsFragment extends Fragment {
+
+    private FragmentHangoutDetailsBinding binding;
+    private Hangout hangout;
+
+    private TextView tvHangoutDetailsAlias;
+    private TextView tvHangoutDetailsLocation;
+    private TextView tvHangoutDetailsDate;
+    private TextView tvHangoutDetailsTime;
+    private TextView tvHangoutDetailsHangoutCode;
+    private ImageButton btnCopyClipboard;
+
+    public HangoutDetailsFragment() {
+    }
+
+    public static HangoutDetailsFragment newInstance(Hangout hangout) {
+        HangoutDetailsFragment fragment = new HangoutDetailsFragment();
+        Bundle args = new Bundle();
+        args.putParcelable(Hangout.FRAGMENT_ARGUMENT_ID, hangout);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (!getArguments().isEmpty()) {
+            hangout = getArguments().getParcelable(Hangout.FRAGMENT_ARGUMENT_ID);
+        }
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        binding = FragmentHangoutDetailsBinding.inflate(getLayoutInflater());
+        return binding.getRoot();
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        btnCopyClipboard = binding.btnCopyClipboard;
+        tvHangoutDetailsDate = binding.tvHangoutDetailsDate;
+        tvHangoutDetailsTime = binding.tvHangoutDetailsTime;
+        tvHangoutDetailsAlias = binding.tvHangoutDetailsAlias;
+        tvHangoutDetailsLocation = binding.tvHangoutDetailsLocation;
+        tvHangoutDetailsHangoutCode = binding.tvHangoutDetailsHangoutCode;
+        setDateTimeTextViews();
+        tvHangoutDetailsAlias.setText(hangout.getAlias());
+        tvHangoutDetailsHangoutCode.setText(hangout.getObjectId());
+        btnCopyClipboard.setOnClickListener(this::onClipboardClick);
+        tvHangoutDetailsLocation.setText(hangout.getLocationString());
+        initMapFragment();
+    }
+
+    private void onClipboardClick(View view) {
+        ClipboardManager clipboard =
+                (ClipboardManager) getActivity()
+                .getSystemService(Context.CLIPBOARD_SERVICE);
+        ClipData clip = ClipData.newPlainText("Hangout Code", hangout.getObjectId());
+        clipboard.setPrimaryClip(clip);
+        Toast.makeText(getContext(), "Copied to clipboard", Toast.LENGTH_SHORT).show();
+    }
+
+    private void setDateTimeTextViews() {
+        Date deadline = hangout.getDeadline();
+        String dateFormat = "MM/dd/yy";
+        String timeFormat = "HH:mm";
```

Comment:
These are same time constants.  Seeing duplicate code is a a chance to consolidate and dedup.  At minimum make these public constants.  You could also consider a DateTimeUtil class that encapsulates related code.